### PR TITLE
More tests from Pygments

### LINKED ISSUE #4: Add more tests from Pygments
Currently we have tests to parse expected token types. Need to add more test cases from https://github.com/pygments/pygments/blob/master/tests/test_data.py

### PR DESCRIPTION
**Pull request by optikfluffel**
There's still enough stuff to do before this can close #4, but it's a start.
Feedback, especially for those TODOs and skipped tests, is very welcome.

### CODE REVIEW

**Comment by lkarthee** [comment]
@optikfluffel Thanks for  the PR. I will look at the TODOs and update tomorrow .

**Comment by optikfluffel** [comment]
@lkarthee Take your time, I'm not in a hurry :)

**Comment by lkarthee** [comment]
JSON-LD is not supported by this lexer. I will not be looking into them as they are skipped.

File: test/makeup_json_test.exs [hunk 1]
```diff
@@ -2,101 +2,215 @@ defmodule MakeupJsonTest do
...
+
+  test "Unfinished or unclosed single-line comments are parsed as errors" do
+    assert [{:error, _, ?/}] = JsonLexer.lex("/")
+    assert [_number, {:error, _, ?/}] = JsonLexer.lex("1/")
+    assert [{:error, _, ?/}, _number] = JsonLexer.lex("/1")
+    assert [_string, {:error, _, ?/}] = JsonLexer.lex("\"\"/")
+  end
+
+  # TODO:
+  @tag :skip
```
Comment (lkarthee) [inline test/makeup_json_test.exs:207]: for multi line comment were incorrectly handled. I have a pending PR. Once PR is merged, I think we can remove skip tag.

**Comment by lkarthee** [comment]
Hi @optikfluffel , I am so sorry I could not review this PR earlier. While looking at your pr - i realised there was a bug regarding multiline comments. Thank you.

Token output of makeup lexer does not match the pygments lexer exactly. But the tag/token_type will be equivalent so that highlighting will be matching the pygments output.

For escape sequences test cases - we have to look at the tokens and write assert statements accordingly.

File: test/makeup_json_test.exs (hunk 1 above)
Comment (lkarthee) [reply test/makeup_json_test.exs:207]: PR is merged. :skip tag can be removed.

**Pull request merged**